web/src/app/admin/skills/skills.component.ts

```diff
@@ -1,14 +1,105 @@
-import {Component, OnInit} from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { SkillsService } from './shared/skills.serviceApp';
```

Comment:
change SkillsService name to SkillsServiceApp every where

Reply:
done